lib/actions.js

```diff
@@ -332,15 +332,26 @@ actions.ChangeSetNotExecutableError = error('ChangeSetNotExecutableError');
  * @param {function} callback - a function that will be provided with the changeset
  * details once the changeset is in CREATE_COMPLETE, DELETE_COMPLETE, or FAILED state
  */
-function describeChangeset(cfn, name, changesetId, callback) {
-  cfn.describeChangeSet({ ChangeSetName: changesetId, StackName: name }, function(err, data) {
-    if (err) return callback(err);
-
-    if (data.Status === 'CREATE_COMPLETE' || data.Status === 'FAILED' || data.status === 'DELETE_COMPLETE')
-      return callback(null, data);
-
-    setTimeout(describeChangeset, 1000, cfn, name, changesetId, callback);
-  });
+function describeChangeset(cfn, name, changesetId, nextToken, callback) {
```

Comment:
The `nextToken` can be removed here, since it's only needed when using `callAPI` below